src/mgr/PyModuleRegistry.h

```diff
@@ -227,5 +227,15 @@ class PyModuleRegistry
     return v;
   }
 
+  bool is_module_active(const std::string &name) {
+    ceph_assert(active_modules);
+    return active_modules->module_exists(name);
+  }
+
+  auto& get_active_module_finisher(const std::string &name) {
```

Comment:
```suggestion
  auto& get_active_module_finisher(std::string_view name) {
```

Reply:
The same as above. It needs to be converted back to std::string.